rest/taskana-rest-spring-example-boot/src/main/java/pro/taskana/rest/security/SpringBootWebSecurityConfig.java

```diff
@@ -0,0 +1,180 @@
+package pro.taskana.rest.security;
```

Comment:
I've never done something with websecurityconfig. 
Is something special done here?